vm/src/lib.rs

```diff
@@ -63,5 +62,8 @@ pub mod types;
 pub mod utils;
 pub mod vm;
 
+// This is to make my life easier for now, will move to Felt soon
+pub use stark_felt::Felt as Felt252;
```

Comment:
Can we address this now, or leave a TODO so we don't miss it?

Reply:
I'm inclined to put a TODO. The PR is big as it is.

Reply:
It can be an immediate follow-up.